go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Scalingo/gopassword
 
-go 1.20
+go 1.23
```

Comment:
issue: Go 1.23 is not installed on the servers. Only Go 1.22 (https://github.com/Scalingo/appsdeck-kitchen/blob/7b6e1f5f12eba1d3f24f6da13214cf1dc5874254/environments/production.rb#L15-L18). Hence I would only update this parameter to 1.22